src/zoid/buttons/component.jsx

```diff
@@ -235,6 +238,19 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 }
             },
 
+            onInit: {
+                type:     'function',
+                required: false,
+                default:  () => noop,
+                decorate: ({ props, value = noop }) => {
+                    return (...args) => {
+                        enableVenmoExperiment.logStart({ [ FPTI_KEY.BUTTON_SESSION_UID ]: props.buttonSessionID });
```

Comment:
I noticed an older experiment using onInit this way to emit the logStart event. Is this needed?

Reply:
Yep, this is needed